tests/integration/test_extract.py

```diff
@@ -179,6 +179,97 @@ def test_place_basic(shape, vals):
     assert np.array_equal(arr_np, arr_num)
 
 
+@pytest.mark.xfail(reason="cunumeric raises exception when vals is ndim")
+@pytest.mark.parametrize("vals", VALUES, ids=str)
+@pytest.mark.parametrize("ndim", range(2, DIM), ids=str)
+def test_place_vals_ndim(vals, ndim):
+    shape = (1, 2, 3)
+
+    arr_np = np.random.random(shape)
+    arr_num = num.array(arr_np)
+    mask_np = np.random.random(size=arr_np.size).astype(bool)
```

Comment:
same question re: use of randomness. Usually we will want test runs to be predictable and uniform unless there is something specific to test in relation to randomness